src/main/java/de/hpi/bpt/argos/util/threading/BackgroundWorkerImpl.java

```diff
@@ -0,0 +1,56 @@
+package de.hpi.bpt.argos.util.threading;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Consumer;
+
+/**
+ * {@inheritDoc}
+ * This is the implementation.
+ */
+public class BackgroundWorkerImpl<DataType> implements BackgroundWorker<DataType> {
+	private static final Logger logger = LoggerFactory.getLogger(BackgroundWorkerImpl.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "logger" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1068)